front/src/modules/infra/components/InfraSelector/InfraSelectorModalBodyStandard.tsx

```diff
@@ -49,6 +54,20 @@ export default function InfraSelectorModalBodyStandard({
   const { closeModal } = useContext(ModalContext);
   const navigate = useNavigate();
 
+  const { data: subjectsList } = mockedEditoastApi.endpoints.getSubjects.useQuery();
```

Comment:
Hum I think the subject are only used to get the users. Can you create a `users` const to extract directly the users from `subjectsList` ?